kitchen-sink-demo/src/main.augury.ts

```diff
@@ -0,0 +1,21 @@
+import { enableProdMode, NgZone } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { AppModule } from './app/app.module';
+import { environment } from './environments/environment';
+
+// @todo: how to not include in prod bundle?
+import { auguryBootstrap } from '../../../augury-labs/packages/core/dist'; // '@augury/core';
```

Comment:
We should look at using optional npm packages instead of using relative paths here.